examples/ExampleApp.py

```diff
@@ -318,6 +318,7 @@ def __init__(self):
 
         textFil = self.ui.exampleFilter
         self.curListener = None
+        self.ui.exampleFilter.setFocus()
```

Comment:
The entire goal of this PR is to be able to start typing to filter as soon as you open the example app (without having to click-focus the search bar)